docs/architecture/adr-015-ibc-packet-receiver.md

```diff
@@ -0,0 +1,105 @@
+# ADR 015: IBC Packet Receiver
+
+## Changelog
+
+- 2019 Oct 22: Initial Draft
+
+## Context
+
+[ICS 26](https://github.com/cosmos/ics/tree/master/spec/ics-026-routing-module) defines function `handlePacketRecv`. 
+`handlePacketRecv` executes per-module `onRecvPacket`, verifies the packet merkle proof, and pushes the acknowledgement bytes
+to the state. 
```

Comment:
which state? Which store key does would it use? would this require another Keeper?

Reply:
The acknowledgements are pushed into the IBC keeper state

Reply:
which of all the IBC keepers? the channel keeper (ICS04) or the transfer keeper (ICS20)

Reply:
The channel keeper(under `ports/portID/channels/chanID/acknowledgement/sequence`)